tests/test_globals.hpp

```diff
@@ -31,10 +31,11 @@ struct print_log_value<glm::vec3> {
         s << glm::to_string(v);
     }
 };
+BOOST_NS_MAGIC_CLOSING
 }
 }
-BOOST_NS_MAGIC_CLOSING
 
+#if BOOST_VERSION < 106400
```

Comment:
I'm not surewhich version added the nullptr_t specialization. It might be 1.63.0 (or lower).